demo/src/index.js

```diff
@@ -8,7 +8,7 @@ import iiifManifest from '../../src/json/manifest-pawpaw-mahler';
 
 // A valid IIIF Manifest URI endpoint
 const iiifManifestUrl =
-  'https://pawpaw.dlib.indiana.edu/media_objects/2j62s484w/manifest.json';
+  'http://dlib.indiana.edu/iiif_av/mahler-symphony-3/mahler-symphony-3.json';
```

Comment:
Chrome has started prohibiting mixed content requests so this might not work if it's used in a https website. I'd change it to https to avoid potential problems.